examples/bert/README.md

```diff
@@ -267,3 +267,22 @@ python bert_classifier_adaptive.py --do-train --do-eval \
 ```
 See [here](https://adaptdl.readthedocs.io/en/latest/standalone-training.html)
 for full documentation on how to train the model in standalone mode.
+
+
+## Run NNI tuners with adaptdl
+A version of the BERT example `bert_classifier_adaptive_nni.py` is provided based on the above example from the adaptive distributed training using AdaptDL, which auto-tunes the hyperparameters specified in `search_space.json`. Accordingly, we prepare a configuration file `config_bert_classifier.yml` for the user to run the experiment with NNI tuners. In this configuration file, everything but several arguments follows the rule of running local NNI experiments as we introduced above. Compared with `config_tuner.yml` we provide for running a local experiment, one difference is that the user needs to specify a `nniManagerIp`, which is an IP address of the machine on which NNI manager process runs and typically on which you submit the job. Also, notice `trainingServicePlatform` is changed to `adl`. The other difference is that the user needs to prepare a docker image to be pulled when submitting the job. In order to build a docker image for the example, the user only needs to go to the root directory of texar and run a docker build command followed by a docker push command. We provide an example about how we build and push a docker image:
+
+```
+docker build -t registry.petuum.com/dev/nni:texar-bert-classifier -f docker/Dockerfile
+```
+
+```
+docker push registry.petuum.com/dev/nni:texar-bert-classifier
+```
+Then in the configuration file, we just need to provide the command line with the code path in the docker image. In addition to these changes, the user needs to set up [AdaptDL](https://adaptdl.readthedocs.io/en/latest/installation/index.html) cluster and install [NNI/petuum-dev](https://github.com/petuum/nni) locally. After all the setups, similarly to running a local job, just run:
```

Comment:
`provide the command line with the code path in the docker image`
This line is slightly hard to understand, could provide an example.

AdaptDL cluster and NNI setup is better explained at the beginning of this documentation session.

Reply:
Have updated the ReadMe file and given an example for this sentence. The NNI setup has already been given at the beginning of the section of NNI with Texar (See: Hyperparamter tuning with Neural Network Intelligence (NNI)). I also added one sentence for AdaptDL setup at the beginning of the section of AdaptDL, and I also suggest the author of that section could add more information about the setup.